nwbinspector/checks/tables.py

```diff
@@ -205,5 +209,5 @@ def check_col_not_nan(table: DynamicTable, nelems: Optional[int] = 200):
             ).flatten()
         ):
             yield InspectorMessage(
-                message=f"Column {column.name} has all NaN values. Consider removing it from the table."
+                message=f"Column '{column.name}' has all NaN values. Consider removing it from the table."
```

Comment:
```suggestion
                message=f"Column '{column.name}' might have all NaN values. Consider removing it from the table."
```

Reply:
what do you think of this? Better yet, we could check for the length of the column. If the column is less than nelems, we can say with certainty that all values are NaNs and if not we can say all values **might** be NaNs

Reply:
Hmm yeah that's true - I can add something like that

Reply:
Added, plus a test for it